company.c

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+
+typedef struct {
+	char name[100];
+	char position[100];
+	int salary;
```

Comment:
salary should be a double! Besides this incorrect format, nicely done. Also, again, more comments though!